Bluewire.Common.Console.UnitTests/Bluewire.Common.Console.UnitTests.csproj

```diff
@@ -34,12 +34,16 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Castle.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=407dd0808d44fbdc, processorArchitecture=MSIL">

```

Comment:
If Moq depends on Castle Core, how come we didn't include references to it before?

Reply:
The old version didn't.

Reply:
Actually the old version did 'depend' on Castle.Core, but it did it by ILmerging the assembly into the Moq assembly instead of referencing a package. This was once the cause of serious confusion when it interacted in weird ways with a 'real' Castle.Core assembly.